Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -32,17 +32,30 @@ public struct Compass: View {
     /// The width and height of the compass.
     private var size: CGFloat = 44
     
+    /// An action to perform when the compass is tapped. Note if `mapViewProxy` is non-`nil`
+    /// this will not be invoked.
```

Comment:
Using the [Note callout](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/Note.html#//apple_ref/doc/uid/TP40016497-CH39-SW1) makes it a little more noticeable in my opinion. Not a big deal either way, take or leave.

```suggestion
    /// An action to perform when the compass is tapped.
    ///
    /// - Note: If `mapViewProxy` is non-`nil` this will not be invoked.
```

Reply:
I removed that sentence completely, as the new initializers will prevent that scenario.